raiden/tests/scenarios/bf7_long_path.yaml

```diff
@@ -0,0 +1,394 @@
+# After opening channels, transfers are made with incrementing length starting with a transfer
+# from node0 to node1 and back, then from node0 to node2 with node1 as a mediatorand back etc.
+# This is done all the way up to a transfer from node0 to node14 and back.
+
+scenario:
+  serial:
+    tasks:
+      - parallel:
+          name: "Open 15 channels in both directions"
+          tasks:
+            - open_channel: {from: 0, to: 1, total_deposit: 1_000_000_000_000_000_000}
+            - open_channel: {from: 1, to: 0, total_deposit: 1_000_000_000_000_000_000}
+            - open_channel: {from: 1, to: 2, total_deposit: 1_000_000_000_000_000_000}
+            - open_channel: {from: 2, to: 1, total_deposit: 1_000_000_000_000_000_000}
+            - open_channel: {from: 2, to: 3, total_deposit: 1_000_000_000_000_000_000}
+            - open_channel: {from: 3, to: 2, total_deposit: 1_000_000_000_000_000_000}
+            - open_channel: {from: 3, to: 4, total_deposit: 1_000_000_000_000_000_000}
+            - open_channel: {from: 4, to: 3, total_deposit: 1_000_000_000_000_000_000}
+            - open_channel: {from: 4, to: 5, total_deposit: 1_000_000_000_000_000_000}
+            - open_channel: {from: 5, to: 4, total_deposit: 1_000_000_000_000_000_000}
+            - open_channel: {from: 5, to: 6, total_deposit: 1_000_000_000_000_000_000}
+            - open_channel: {from: 6, to: 5, total_deposit: 1_000_000_000_000_000_000}
+            - open_channel: {from: 6, to: 7, total_deposit: 1_000_000_000_000_000_000}
+            - open_channel: {from: 7, to: 6, total_deposit: 1_000_000_000_000_000_000}
+            - open_channel: {from: 7, to: 8, total_deposit: 1_000_000_000_000_000_000}
+            - open_channel: {from: 8, to: 7, total_deposit: 1_000_000_000_000_000_000}
+            - open_channel: {from: 8, to: 9, total_deposit: 1_000_000_000_000_000_000}
+            - open_channel: {from: 9, to: 8, total_deposit: 1_000_000_000_000_000_000}
+            - open_channel: {from: 9, to: 10, total_deposit: 1_000_000_000_000_000_000}
+            - open_channel: {from: 10, to: 9, total_deposit: 1_000_000_000_000_000_000}
+            - open_channel: {from: 10, to: 11, total_deposit: 1_000_000_000_000_000_000}
+            - open_channel: {from: 11, to: 10, total_deposit: 1_000_000_000_000_000_000}
+            - open_channel: {from: 11, to: 12, total_deposit: 1_000_000_000_000_000_000}
+            - open_channel: {from: 12, to: 11, total_deposit: 1_000_000_000_000_000_000}
+            - open_channel: {from: 12, to: 13, total_deposit: 1_000_000_000_000_000_000}
+            - open_channel: {from: 13, to: 12, total_deposit: 1_000_000_000_000_000_000}
+            - open_channel: {from: 13, to: 14, total_deposit: 1_000_000_000_000_000_000}
+            - open_channel: {from: 14, to: 13, total_deposit: 1_000_000_000_000_000_000}
+      - parallel:
+          name: "Assert after open"
+          tasks:
```

Comment:
I would add a wait of 10 sec or so. You are going to assert on blockchain states, which might be needed to be confirmed

Reply:
I think it works fine without. For the exact reason you mention. Doing transfers might take some time to finalize, but the blockchain state should not really. We can add a wait though if you want. We haven't run into any errors in this assert though.